.gitignore

```diff
@@ -3,6 +3,9 @@
 /target/
 **/*.rs.bk
 
+# Fuzzing corpuses should be explicitly updated
+fuzz/corpus/
```

Comment:
I guess I don't understand what this means from a philosophical point of view, given that there are files in this repo in that location...